crates/notion-core/src/style.rs

```diff
@@ -47,7 +65,8 @@ pub fn progress_bar(action: &str, details: &str, len: u64) -> ProgressBar {
 
     let bar = ProgressBar::new(len);
 
-    bar.set_message(&format!("{: >12} {}", style(action).green().bold(), details));
+    // Action::MAX_WIDTH = 12
```

Comment:
I think there is a way to pass in an argument for the width after all, something like:

```rust
bar.set_message(&format!("{: >width$} {}", style(action.to_string()).green().bold(), details, width = Action::MAX_WIDTH));
```

I'll do this in a followup commit.

Reply:
(I found this info at:

https://www.reddit.com/r/rust/comments/32wdnj/dynamically_create_a_format_string/

and

https://doc.rust-lang.org/std/fmt/

)